<launch>
	<!-- 集群数量 -->
	<arg name="swarm_num" default="1"/>
    <node pkg="prometheus_simulator_utils" type="fake_odom_node" name="fake_odom_node" output="screen">
        <param name="fake_odom/swarm_num_uav" 			value="$(arg swarm_num)"/>
        <param name="fake_odom/swarm_num_ugv" 			value="0"/>
        <param name="fake_odom/manual_init_pos" 		value="true"/>
        <param name="fake_odom/uav1_init_x" 			value="4.0"/>
        <param name="fake_odom/uav1_init_y" 			value="0.0"/>
        <param name="fake_odom/uav1_init_z" 			value="0.0"/>
        <param name="fake_odom/uav1_init_yaw" 			value="0.0"/>
        <param name="fake_odom/pub_gazebo_model_state" 	value="true"/>
    </node>
</launch>


